lib/node-package-manager.ts

```diff
@@ -120,7 +120,8 @@ export class NodePackageManager implements INodePackageManager {
 	}
 
 	public async getRegistryPackageData(packageName: string): Promise<any> {
-		const url = `https://registry.npmjs.org/${packageName}`;
+		const registry = await this.$childProcess.exec(`npm config get registry`);
```

Comment:
What do you think about falling back to registry.npmjs.org in case `npm config get registry` fails for some reason?
For example:
```TypeScript
let registry = "https://registry.npmjs.org/";
try {
    registry = await this.$childProcess.exec(`npm config get registry`);
} catch (err) {
    this.$logger.trace("Unable to get information for current npm registry", err);
}

const url = registry.trim() + packageName;
```

Reply:
I thought about this as well - defensive programming FTW ;-) But even if I wipe my `.npmrc` file, the value npm defaults to is always `https://registry.npmjs.org/`

Reply:
Ok, thanks for the explanation. We'll run our tests now and if the build is green, I'll merge this PR.